src/sonic-bgpcfgd/bgpcfgd/main.py

```diff
@@ -25,7 +25,7 @@
 def do_work():
     """ Main function """
     frr = FRR(["bgpd", "zebra", "staticd"])
-    frr.wait_for_daemons(seconds=20)
+    frr.wait_for_daemons(seconds=60)
```

Comment:
Do we really need to wait for this long? Is there a scenario that the 20-second is not enough?